metagraph/src/cli/align.cpp

```diff
@@ -74,13 +77,25 @@ DBGAlignerConfig initialize_aligner_config(const Config &config) {
     return aligner_config;
 }
 
+template <class Graph, template <class ... Types> class Aligner>
+inline std::unique_ptr<graph::align::IDBGAligner>
+build_aligner_infer_args(const Graph &graph, const DBGAlignerConfig &aligner_config) {
+    return std::make_unique<Aligner<>>(graph, aligner_config);
+}
```

Comment:
Why is this needed? Can't you just write `std::make_unique<...>(graph, aligner_config);`?

Reply:
Actually, there's a mistake here, I'm not passing template arguments for the Aligner construction...

Anyway, this is needed because `ILabeledAligner` takes an `AnnotatedDBG` as input instead of `DeBruijnGraph`

Reply:
Hm, I still don't see why you need this function.

Reply:
I ended up removing it...I actually don't remember why I needed the helper...